policyengine_canada/variables/gov/provinces/bc/benefits/bc_benefits.py

```diff
@@ -0,0 +1,12 @@
+from policyengine_canada.model_api import *
+
+
+class bc_benefits(Variable):
+    value_type = float
+    entity = Household
+    label = "BC benefits"
+    documentation = "British Columbia benefits"
+    unit = CAD
+    definition_period = YEAR
+
+    adds = "gov.province.bc.benfits.benefits"
```

Comment:
That this didn't break it suggests it's not being fully tested. Could you add an integration test? I'm not sure the empty list parameter will work.
```suggestion
    adds = "gov.province.bc.benefits.benefits"
```